.github/workflows/ci.yml

```diff
@@ -8,6 +8,8 @@ on:
     branches:
       - main
       - opendistro-*
+env:
```

Comment:
nit: It will be good to add an empty line before `env` to be consistent with the format in this file.

Reply:
done